carousel/docs/tutorials/tutorial_2.rst

```diff
@@ -138,32 +138,38 @@ Calculations can also be specified in a parameter file. For example copy the
 following into ``PVPower/calculations/utils.json``::
 
     {
-      "dependencies": ["PerformanceCalcs"],
-      "static": [
-        {
+      energy = {
```

Comment:
* This isn't valid JSON,
  - keys must be a string, and strings use double quotes only, therefore this key would be "energy"
  - and keys must be separated from their value by a colon, ":", you cannot use an equal sign here
* if you tried to parse this using [the standard Python json module](https://docs.python.org/2/library/json.html) it would raise an exception
* see [JSON docs](http://json.org/) for what can be used in "JavaScript Object Notation".